Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/App/Server.hs

```diff
@@ -30,7 +30,7 @@ run :: Env -> Application
 run = withModifiedEnv $ \modifiedEnv ->
   BU.run driverOfferAPI driverOfferServer context modifiedEnv
     & logRequestAndResponse modifiedEnv
-    -- & logBecknRequest modifiedEnv
+    & logBecknRequest modifiedEnv
```

Comment:
Should I uncomment this? If yes, then we will store beckn-requests to DB as it was earlier
Maybe we should add some setting for pushing in Kafka, but not store to DB?